joomla/joomla-cms
# [4.3][Guided Tours] Plugin registers events only under administrator

### Summary of Changes

The plugin registers events only when under administrator.

### Testing Instructions

Run the patch and see if the tours are still available and able to run.
There is no longer a load in the site side.

### Actual result BEFORE applying this Pull Request

The tours are available and can run.

### Expected result AFTER applying this Pull Request

The tours are available and can run.

### Link to documentations
Please select:
